src/mergeMockHappData.js

```diff
@@ -0,0 +1,47 @@
+// This is temporary for version 0.1.0 to provide data that will come from hpos, service logger and holofuel in later versions.
```

Comment:
The reason it's up here is because it's very temporary and I don't want it to get cozy. I'd like to leave it here, so that we keep bumping into it and saying: "can we get rid of this yet?"